packages/react/src/NavList/__snapshots__/NavList.test.tsx.snap

```diff
@@ -339,7 +339,7 @@ exports[`NavList renders a simple list 1`] = `
       >
         <a
           aria-current="page"
-          aria-labelledby=":r2:--label "
+          aria-labelledby=":r2:--label  "
```

Comment:
Could me make the filtering string on aria-labelledby as well so that we don't get these extra spaces 😅

Reply:
Yes, definitely! I forgot I turned on auto merge 🤣, I can follow up in one of the PRs that I'll make for one of [the sev-3 issues](https://github.com/github/primer/issues/3558)!